include/rtcdcpp/NiceWrapper.hpp

```diff
@@ -37,8 +37,6 @@
 #include <memory>
 #include <string>
 
-#include <json/json.h>
```

Comment:
I get the changes; now that librtcdcpp is trying to match more closely with other WebRTC implementations, having this api accept JSON no longer makes sense.